fili-navi/src/main/java/com/yahoo/bard/webservice/data/dimension/Tag.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.dimension;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * An interface for tag.
```

Comment:
I was thinking maybe one day we might need tag for other things, so this serves as a general purpose tag with a name interface. Let me know if you think we should indicate that it is for dimension field for now.